ideascube/serveradmin/tests/__init__.py

```diff
@@ -0,0 +1,59 @@
+try:
+    from NetworkManager import NM_DEVICE_TYPE_ETHERNET, NM_DEVICE_TYPE_WIFI
+
+except Exception:
+    # Use hard-coded values for the tests, these are constants anyway
+    NM_DEVICE_TYPE_ETHERNET = 1
+    NM_DEVICE_TYPE_WIFI = 2
+
+
+class NMAccessPoint(object):
+    def __init__(self, ssid, strength, wpa_flags):
+        self.Ssid = ssid
```

Comment:
Humm, not sure about using capitalized properties. I guess it's for consistency with NetworkManager conventions, but then we are hitting python itself (and ideascube) conventions as you know, and I'd better arbitrate for the later :)
(Eg. https://github.com/ideascube/ideascube/pull/197/files#diff-7f6f126831854a0d948da9d936257df2R27 ;) )